Scripts/DCS-BIOS/lib/modules/aircraft_modules/C-130J.lua

```diff
@@ -222,6 +222,15 @@ local devices = {
 -- Center Console
 
 -- Pilot Lighting Panel
+local PLT_LIGHTING_PANEL = "Pilot Lighting Panel"
+C_130J:define3PosTumb("PLT_MASTER_LIGHTING_MODE", devices.LIGHTING_PANELS, 3029, 1337, PLT_LIGHTING_PANEL, "Lighting Mode Master Switch")
+C_130J:defineSpringloaded_3PosTumb("PLT_ANNUNCIATOR_BRIGHTNESS", devices.LIGHTING_PANELS, 3030, 3030, 1336, PLT_LIGHTING_PANEL, "Annunciator Light Brightness Switch")
+C_130J:definePotentiometer("PLT_DOME_LIGHT_BRIGHTNESS", devices.LIGHTING_PANELS, 3017, 1340, { 0, 1 }, PLT_LIGHTING_PANEL, "Cockpit Dome Lighting Brightness Knob")
+C_130J:definePotentiometer("PLT_CB_LIGHT_BRIGHTNESS", devices.LIGHTING_PANELS, 3018, 1341, { 0, 1 }, PLT_LIGHTING_PANEL, "Pilot Circuit Breaker Lighting Brightness Knob")
```

Comment:
Ideally the naming here is consistent. I assume the dome light brightness knob only controls the pilot's dome light, right? If that's the case, it's probably worth including "Pilot" in the name just like you did for the circuit breaker control. The same goes for the other switches.

I could also see an argument that because they're on the pilot panel, they don't need Pilot in the name. I'm fine with that too - just ideally we're consistent.